external-crates/move/Cargo.toml

```diff
@@ -118,7 +118,7 @@ tera = "1.16.0"
 thiserror = "1.0.24"
 tiny-keccak = { version = "2.0.2", features = ["sha3"] }
 tokio = { version = "1.18.2", features = ["full"] }
-toml = "0.5.8"
+toml = {version = "0.5.8", features = ["preserve_order"]}
```

Comment:
Do we need preserve order any more?
Can we just "keyword" `local`, `git` and `r.<ident>`?

Reply:
We can, although we'd then need to scan each field. But that doesn't sound too bad...

Reply:
Removed! 🎉